src/libutil/unix/file-descriptor.cc

```diff
@@ -120,14 +120,35 @@ void Pipe::create()
 
 //////////////////////////////////////////////////////////////////////
 
-void unix::closeMostFDs(const std::set<int> & exceptions)
+#if __linux__ or __FreeBSD__
```

Comment:
```suggestion
#if __linux__ || __FreeBSD__
```
The alternate operator syntax is pretty obscure, I've never seen it used before.